src/blocked.c

```diff
@@ -110,6 +114,12 @@ void processUnblockedClients(void) {
          * client is not blocked before to proceed, but things may change and
          * the code is conceptually more correct this way. */
         if (!(c->flags & CLIENT_BLOCKED)) {
```

Comment:
I don't really follow this comment.

Reply:
The client was blocked.  It was then unblocked and put onto the unblocked list.  Now we are processing that list...

At this point, how can CLIENT_BLOCKED be set?  If it's set, we shouldn't have moved it to the unblocked list, right?

I agree that it can be set AFTER calling `processCommandAndResetClient`, but I'm not sure how the client can be blocked at this line.

Reply:
Oh, I think the comment "the code is conceptually more correct this way" is the only reason this is there.